packages/gp2-components/src/organisms/__tests__/UserProjects.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { gp2 } from '@asap-hub/model';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import UserProjects from '../UserProjects';
+
+describe('UserProjects', () => {
+  type Project = gp2.UserResponse['projects'][number];
+  const getProjects = (length = 1): Project[] =>
+    Array.from({ length }, (_, itemIndex) => ({
+      id: `id-${itemIndex}`,
+      title: `a title ${itemIndex}`,
+      members: [],
+      status: 'Active',
+    }));
+  const firstName: gp2.UserResponse['firstName'] = 'John';
+  const id = 'user-id';
+  const renderUserProjects = (projects: Project[]) =>
+    render(<UserProjects projects={projects} firstName={firstName} id={id} />);
+  it('renders the short text when there are no projects', () => {
+    renderUserProjects([]);
+    expect(
+      screen.getByText(`${firstName} has been involved`, { exact: false }),
+    ).toBeInTheDocument();
```

Comment:
recently we've moved to toBeVisible when checking for existence on the page. 

When we check the element is not there we use not.toBeInTheDocument.